chart/templates/deployment.yaml

```diff
@@ -82,90 +82,39 @@ spec:
               value: '{{ ((.Values.prometheus).createTargets) | default "true" }}'
             - name: CREATE_ALERTS
               value: '{{ ((.Values.prometheus).createAlerts) | default "true" }}'
-            - name: K8S_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
-        - name: distributor
-          image: "{{ .Values.distributor.image.repository }}:{{ .Values.distributor.image.tag | default .Chart.AppVersion }}"
-          livenessProbe:
-            httpGet:
-              path: /health
-              port: 8080
-            initialDelaySeconds: 0
-            periodSeconds: 5
-          readinessProbe:
-            httpGet:
-              path: /health
-              port: 8080
-            initialDelaySeconds: 5
-            periodSeconds: 5
-          imagePullPolicy: {{ .Values.distributor.image.pullPolicy }}
-          ports:
-            - containerPort: 8080
-          resources:
-            requests:
-              memory: "16Mi"
-              cpu: "25m"
-            limits:
-              memory: "32Mi"
-              cpu: "100m"
-          env:
-            - name: PUBSUB_URL
-              value: 'nats://keptn-nats'
             - name: PUBSUB_TOPIC
-              value: 'sh.keptn.event.monitoring.configure,sh.keptn.event.configure-monitoring.triggered,sh.keptn.event.get-sli.triggered'
-            - name: PUBSUB_RECIPIENT
-              value: '127.0.0.1'
-            - name: PUBSUB_RECIPIENT_PATH
-              value: '/events'
-            - name: STAGE_FILTER
-              value: "{{ .Values.distributor.stageFilter }}"
-            - name: PROJECT_FILTER
-              value: "{{ .Values.distributor.projectFilter }}"
-            - name: SERVICE_FILTER
-              value: "{{ .Values.distributor.serviceFilter }}"
-            - name: DISTRIBUTOR_VERSION
-              value: {{ .Values.distributor.image.tag | default .Chart.AppVersion }}
-            - name: VERSION
+              value: {{ ((.Values).subscription).pubsubTopic | default "sh.keptn.>" }}
```

Comment:
not sure if this default value still works with the go-sdk

Reply:
I remember @bacherfl mentioning that `sh.keptn.>` still works.